### PR TITLE
docs: Websockets

### PR DESCRIPTION
**Pull request by tjholm**
depends on: https://github.com/nitrictech/node-sdk/pull/196

### CODE REVIEW

File: src/pages/websockets.mdx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+socket.on('disconnect', async (ctx) => {
+  await connections.doc(ctx.req.connectionId).delete()
+})
+
+const broadcast = async (data: string | Uint8Array, as: string = 'server') => {
+  const connectionStream = connections.query().stream()
+
+  const streamEnd =
+    new Promise() <
```
Comment (davemooreuws) [inline src/pages/websockets.mdx:104]: Is this a typo?

**Review by davemooreuws — CHANGES_REQUESTED** [review]
A question and perhaps add some next steps at the end.

File: src/pages/websockets.mdx (hunk 1 above)
Comment (tjholm) [reply src/pages/websockets.mdx]: fixed it, but then re-ran format:fix, this is prettiers idea of better formatting :rofl: 
image

Comment (tjholm) [reply src/pages/websockets.mdx]: was a typo introduced by prettier

**Review by davemooreuws — APPROVED** [review]

File: src/pages/websockets.mdx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    // store any metadata related to the connection here
+    connectionId: ctx.req.connectionId,
+  })
+})
+
+socket.on('disconnect', async (ctx) => {
+  await connections.doc(ctx.req.connectionId).delete()
+})
+
+const broadcast = async (data: string | Uint8Array, as: string = 'server') => {
```
Comment (HomelessDinosaur) [inline src/pages/websockets.mdx:100]: is `as` used?

File: src/pages/websockets.mdx [hunk 3]
```diff
@@ -0,0 +1,124 @@
+export const description = 'Connect and manage websockets with Nitric'
+
+# Websockets
+
+Nitric provides support for serverless websockets. Allowing you to connect client applications to your nitric functions using websocket gateways such as AWS APIGateway.
```
Comment (HomelessDinosaur) [inline src/pages/websockets.mdx:5]:
```suggestion
Nitric provides support for serverless websockets. Allowing you to connect client applications to your Nitric functions using websocket gateways such as AWS APIGateway.
```

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**